### ISSUE TITLE
Dump stack action is not working on Pharo 7

### ISSUE THREAD

**Issue by gcotelli** [ba-st/ApplicationStarter#5]
https://travis-ci.org/ba-st/ApplicationStarter/jobs/467208793

**Comment by gcotelli** [comment]
I've started to do a fix but noticed: pharo-project/pharo#2335.
Until it's not fixed the test will not work.

**Comment by gcotelli** [comment]
When the related bug is fixed, we can change:

```smalltalk
dumpStackAndReport: exception

	^ self
		logError: ( 'Dumping Stack Due to Unexpected Error: <1s>' expandMacrosWith: exception messageText )
		during: [ | serializer |

			serializer := FLSerializer newDefault.
			serializer analyzer when: [ :object | object class isObsolete ] substituteBy: [ :object | nil ].
			serializer
				addPostMaterializationAction: [ :materialization | 
					Smalltalk tools debugger
						openOn: ( Processor activeProcess newDebugSessionNamed: 'External stack' startedAt: materialization root )
						withFullView: true
					].
				].
			self defaultStackDumpFile binaryWriteStreamDo: [ :stream |serializer serialize: thisContext on: stream ]
			]
```
